⚠️DO NOT MERGE⚠️: Add paraview+qt to Windows gitlab CI

johnwparent opened the pull request:
Disables MPI support for paraview as that was previously causing issues with Windows CI.

Currently this branch wants to move faster than PR fixes for broken things on Windows in develop can land, so this PR contains changes from a number of other patch PRs. Only the HEAD commit of this PR is what the final change set should look like. If this PR doesn't only touch the pipeline file to add paraview, don't merge it.

Cannot be draft as we need gitlab testing.

johnwparent committed:
Add paraview+qt to Windows gitlab CI

kwryankrattiger commented on share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml:
```diff
@@ -141,7 +141,7 @@ default:
       --config-scope "${SPACK_CI_CONFIG_ROOT}/${SPACK_TARGET_PLATFORM}/${SPACK_TARGET_ARCH}"
       config blame > "${CI_PROJECT_DIR}/jobs_scratch_dir/spack.yaml.blame"
-    - spack -v  --color=always
+    - spack -dv  --color=always
```
Do not merge until `-d` is dropped

johnwparent committed:
Python: pyconfig.h changed locations in 3.13 on Windows

johnwparent committed:
Make Windows make tools make executables

Adds a setup method to the msvc compiler class to effect this once compilers as nodes lands

johnwparent commented:
@spackbot fix style

johnwparent committed:
ICU4C: Ensure no doubly quoted args

ICU4C's NMAKE seems to over-quote to the degree
that it  passes paths like ""<path>"" which
confuses the Python command line in subprocesses
the build starts
